stress/common/src/main/java/alluxio/stress/fuse/FUSEBenchOperation.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.fuse;
+
+import alluxio.stress.client.ClientIOOperation;
+
+public enum FUSEBenchOperation {
```

Comment:
Same as other fuse classes, we use `Fuse` instead of `FUSE` in code.
Would prefer to name those classes FuseOperation, FuseParameters..... 
no need to add bench (short and clear name)

Reply:
Actually we can use FuseIO instead of FuseBench
same for all other classes